android/app/src/main/java/com/expensify/chat/bootsplash/BootSplashActivity.java

```diff
@@ -0,0 +1,42 @@
+package com.expensify.chat.bootsplash;
+
+import android.content.Intent;
+import android.os.Bundle;
+
+import androidx.annotation.Nullable;
+import androidx.appcompat.app.AppCompatActivity;
+
+import com.expensify.chat.MainActivity;
+
+public class BootSplashActivity extends AppCompatActivity {
+
+  protected void forwardIntentToMainActivity(Intent intent) {
+    Intent intentCopy = new Intent(intent);
```

Comment:
As this line only copies the instructor, I think cloning the intent would be simpler. Then we would just need to change the class to `MainActivity`.

`Intent intentCopy = intent.clone()`

Reply:
You are right, I will update this.